### PR TITLE
Copies template to user land.

### LINKED ISSUE #428: Write `template.html` to Project
**Do you want to request a *feature* or report a *bug*?**

Feature

**If this is a feature request, what is motivation or use case for changing the behavior?**

In order to make any customizations to the `lib/template.html`, one has to manually copy & paste the contents into the current directory.

I have two suggestions, an either/or scenario:

1) Write the `lib/template.html` to the project by default during `create`
2) Add some kind of command or flag to print the current template to project.
    _This could go nicely with #427 + its duplicate requests... write all internal configs to repo?_

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #428 

**What kind of change does this PR introduce?**
Copies `template.html` to user land.

**Did you add tests for your changes?**
Yes

**Summary**
Copies `template.html` to user land at the time of creation and uses it from there.

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: packages/cli/lib/lib/webpack/render-html-plugin.js [hunk 1]
```diff
@@ -6,11 +6,14 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { readJson } = require('./webpack-base-config');
 const prerender = require('./prerender');
 const createLoadManifest = require('./create-load-manifest');
-const template = resolve(__dirname, '../../resources/template.html');
+let template = resolve(__dirname, '../../resources/template.html');
 
 module.exports = function(config) {
 	const { cwd, dest, isProd, src } = config;
-
+	console.log(resolve(cwd, dest, 'template.html'));
```
Comment (ForsakenHarmony) [inline packages/cli/lib/lib/webpack/render-html-plugin.js]: you'll probably want to remove this before the merge 🙃

**Comment by ForsakenHarmony** [comment]
you're gonna have to adjust the test snapshots for the new files

**Comment by prateekbh** [comment]
@ForsakenHarmony When building a new project with current code and trying to build it I am getting
```
Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
```

This is stopping me from proceeding, any idea why this is happening?

**Comment by ForsakenHarmony** [comment]
maybe remove all the `node_modules` folders and run `yarn` in the root dir

**Comment by prateekbh** [comment]
still same

screen shot 2019-01-23 at 10 14 37 am

**Comment by ForsakenHarmony** [comment]
does that temp project have babel 6 as a dependency anywhere?

**Comment by prateekbh** [comment]
yes there is and i dont know where it is coming from https://filebin.net/8ccgy99jdr325eu3

**Comment by ForsakenHarmony** [comment]
you seem to have an old local version of `preact-cli` in that project

**Comment by prateekbh** [comment]
@ForsakenHarmony  @developit @lukeed this PR just copies the current template to user land. Should we get this out and re-visit making `template.html` better in 4.0?

**Comment by ForsakenHarmony** [comment]
probably

**Comment by prateekbh** [comment]
Will merge this PR EOD today

**Pull request merged**